built_value/lib/src/built_set_multimap_serializer.dart

```diff
@@ -62,11 +62,10 @@ class BuiltSetMultimapSerializer
     for (int i = 0; i != serialized.length; i += 2) {
       final key = serializers.deserialize(serialized.elementAt(i),
           specifiedType: keyType);
-      result.addValues(
-          key,
-          serialized.elementAt(i + 1).map((value) =>
-                  serializers.deserialize(value, specifiedType: valueType))
-              as Iterable);
+      for (final value in serialized.elementAt(i + 1).map((value) =>
+          serializers.deserialize(value, specifiedType: valueType))) {
```

Comment:
I'd extract this function out of the for,  too complex.

Reply:
Done.